test/test_respondent_controller.py

```diff
@@ -1494,11 +1494,11 @@ def test_put_change_respondent_account_status_bad_oauth2_response(self):
 
     def test_change_respondent_password_bad_auth_response(self):
         with patch('ras_party.controllers.account_controller.decode_email_token'), \
-             patch('ras_party.controllers.account_controller.current_app'), \
-             patch('ras_party.support.session_decorator.current_app.db'), \
-             patch('ras_party.controllers.account_controller.enrol_respondent_for_survey'), \
-             patch('ras_party.controllers.account_controller.NotifyGateway'), \
-             patch('ras_party.controllers.account_controller.OauthClient') as auth, \
+                patch('ras_party.controllers.account_controller.current_app'), \
```

Comment:
did you mean to change this indentation?

Reply:
no but the auto format for pycharm did and it looks good now